visualization/tslint.json

```diff
@@ -8,7 +8,7 @@
         "no-eval": true,
         "no-conditional-assignment": true,
         "new-parens": true,
-        "no-unused-variable": true,
+        "no-unused-variable": [true, {"ignore-pattern": "^_"}],
```

Comment:
underscore => variable is used in template and shouldn't be deleted.
make severity to warning.
Conventin should be noted somewhere

Reply:
See [wiki](https://github.com/MaibornWolff/codecharta/wiki/to-be-documented)